e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -909,10 +909,13 @@ describe("scenarios > embedding > full app", () => {
      * This step only shows when there are more than 1 option in the bucket step, the only time this happens
      * is when there are both selectable models and tables for the current user.
      */
-    function goBackToBucketStep() {
+    /** @param {'from-table' | 'from-model'} fromStep */
```

Comment:
since the new behavior is that when joining from a model it will open the model step immediately and the model step only need to go back once to reach the bucket step.